plugins/vli/vli-noinst.quirk

```diff
@@ -113,3 +107,58 @@ VliDeviceKind = VL106
 Plugin = vli
 GType = FuVliPdDevice
 VliDeviceKind = VL107
+
+# Generic devices
+[USB\VID_2109&PID_D101]
+Plugin = vli
+GType = FuVliPdDevice
+[USB\VID_2109&PID_D102]
+Plugin = vli
+GType = FuVliPdDevice
+
+[USB\VID_2109&PID_8880]
+Plugin = vli
+GType = FuVliPdDevice
+
+# VL671,U3TT, VT3547
+[USB\VID_2109&PID_8881]
+Plugin = vli
+GType = FuVliPdDevice
+[USB\VID_2109&PID_8882]
+Plugin = vli
+GType = FuVliPdDevice
+[USB\VID_2109&PID_8883]
+Plugin = vli
+GType = FuVliPdDevice
+
+# Realtek
+[USB\VID_2109&PID_8884]
+Plugin = vli
+GType = FuVliPdDevice
+
+# VL650, VT3555
+[USB\VID_2109&PID_8885]
+Plugin = vli
+GType = FuVliPdDevice
+
+# MStar, VT3518
+[USB\VID_2109&PID_8886]
+Plugin = vli
+GType = FuVliPdDevice
```

Comment:
@memily are we sure none of these are on the LVFS?

Reply:
Yes, these are vids/pids created for designs that never made it to production --ergo that one vid/pid in the Lenovo file...